edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/modules/KubernetesModule.cs

```diff
@@ -111,6 +114,7 @@ public class KubernetesModule : Module
             this.closeOnIdleTimeout = closeOnIdleTimeout;
             this.idleTimeout = idleTimeout;
             this.experimentalFeatures = experimentalFeatures;
+            this.tokenSource = new CancellationTokenSource(SystemInfoTimeout);
```

Comment:
lets make it local variable and move it closer to usage. in future if someone will try to resolve other instance of a DockerEnvironmentProvider it will be a new token for it